mne/report.py

```diff
@@ -800,7 +811,7 @@ def _add_figs_to_section(self, figs, captions, section='custom',
                                              caption=caption,
                                              show=True,
                                              image_format=image_format,
-                                             width=scale, comment=comment)
```

Comment:
I don't think this should be here @dengemann, because in this bit of code the figure has already been scaled e.g. to `0.5` (and this would set the `width=0.5%` in HTML), WDYT?